tests/browserstack_automation/conf/test.conf.js

```diff
@@ -1,6 +1,6 @@
 const parallelConfig = {
-  user: 'josephevans_Tl1dVS',
-  key: 'MBfCTeKv7hYraEHusDcA',
+  user: 'dalemcgrew1',
+  key: '8J93xi57VBQbEBpqdMJR',
```

Comment:
Hi @jeje1197, make sure to move the user and key into the WebApp/tests/browserstack/browserstack.config.js file in your next check in, and then we can reset both of these keys.